server/models/triprequest.js

```diff
@@ -0,0 +1,25 @@
+
+export default (sequelize, DataTypes) => {
```

Comment:
@irfiacre the trip request model/table should have its own id that would be used as a reference. Therefore, add one more field i.e. trip_request_id.

Reply:
I used the default trip provided in the `xxxxxxxxxxxx-create-triprequest` migration as the one for reference for the *trip_request_id*.

Reply:
It's okay.